packages/common/src/registry.ts

```
@@ -0,0 +1,115 @@
+import ProjectRegistryABI from "./abis/ProjectRegistry.json";
+import { ethers } from "ethers";
+import { Provider } from "@wagmi/core";
+import { Web3Provider } from "@ethersproject/providers";
+import {
+  Multicall,
+  ContractCallContext,
+} from "ethereum-multicall";
+
+export const fetchProjectOwners = async (
+  provider: Provider | Web3Provider,
+  chainID: number,
+  projectID: number,
+): Promise<string[]> => {
+  const addresses = addressesByChainID(chainID);
+
+  const projectRegistry = new ethers.Contract(
+    addresses.projectRegistry!,
+    ProjectRegistryABI,
+    provider,
+  );
+
+  try {
+    const projectOwners = await projectRegistry.getProjectOwners(projectID);
+    return projectOwners as string[];
+  } catch (err) {
+    // console.log(err);
+    console.log(
```

shouldn't we just return the error? or at least log it with `console.error`